tests/conftest.py

```diff
@@ -69,12 +88,31 @@ def start_process(path, command):
         return process
 
 
+@pytest.fixture(name="get_experiment_json")
+def fixture_get_experiment_json(request):
+    experiment_slug = request.config.getoption("--experiment-slug")
+    slug_server = request.config.getoption("--experiment-server")
+    experiment_json = request.config.getoption("--experiment-json")
+
+    if experiment_json:
+        with open(experiment_json) as f:
+            return json.load(f)
+    else:
```

Comment:
I'm Python ignorant, but does this need an `else:` or can we avoid the nesting since the `if:` already returns?

Reply:
Good catch, the else isn't explicitly needed